src/odemis/util/test/transform_test.py

```diff
@@ -51,65 +51,65 @@
+    (-ROT90, (1., 1.), 0., -TY, [(0., -1.), (0., -2.), (1., -1.)]),
+    (-ROT90, (1., 1.), 0., TXY, [(2., 3.), (2., 2.), (3., 3.)]),
+    (ROT90, (1., 1.), 0., T0, [(0., 0.), (0., 1.), (-1., 0.)]),
+    (ROT90, (1., 1.), 0., TX, [(1., 0.), (1., 1.), (0., 0.)]),
+    (ROT90, (1., 1.), 0., TY, [(0., 1.), (0., 2.), (-1., 1.)]),
+    (ROT90, (1., 1.), 0., -TX, [(-1., 0.), (-1., 1.), (-2., 0.)]),
+    (ROT90, (1., 1.), 0., -TY, [(0., -1.), (0., 0.), (-1., -1.)]),
+    (ROT90, (1., 1.), 0., TXY, [(2., 3.), (2., 4.), (1., 3.)]),
+    (ROT180, (1., 1.), 0., T0, [(0., 0.), (-1., 0.), (0., -1.)]),
+    (ROT180, (1., 1.), 0., TX, [(1., 0.), (0., 0.), (1., -1.)]),
+    (ROT180, (1., 1.), 0., TY, [(0., 1.), (-1., 1.), (0., 0.)]),
+    (ROT180, (1., 1.), 0., -TX, [(-1., 0.), (-2., 0.), (-1., -1.)]),
+    (ROT180, (1., 1.), 0., -TY, [(0., -1.), (-1., -1.), (0., -2.)]),
+    (ROT180, (1., 1.), 0., TXY, [(2., 3.), (1., 3.), (2., 2.)]),
 ]
 
 SIMILARITY_KNOWN_VALUES = copy.copy(RIGID_KNOWN_VALUES)
 SIMILARITY_KNOWN_VALUES.extend([
     # (rotation, scale, shear, translation, [(x0, y0), (x1, y1), ...])
-    (ROT45, SQ2, 0., T0, [(0., 0.), (1., 1.), (-1., 1.)]),
-    (ROT45, SQ2, 0., TX, [(1., 0.), (2., 1.), (0., 1.)]),
-    (ROT45, SQ2, 0., TY, [(0., 1.), (1., 2.), (-1., 2.)]),
-    (ROT45, SQ2, 0., -TX, [(-1., 0.), (0., 1.), (-2., 1.)]),
-    (ROT45, SQ2, 0., -TY, [(0., -1.), (1., 0.), (-1., 0.)]),
-    (ROT45, SQ2, 0., TXY, [(2., 3.), (3., 4.), (1., 4.)]),
-    (0., 2., 0., T0, [(0., 0.), (2., 0.), (0., 2.)]),
-    (0., 2., 0., TX, [(1., 0.), (3., 0.), (1., 2.)]),
-    (0., 2., 0., TY, [(0., 1.), (2., 1.), (0., 3.)]),
-    (0., 2., 0., -TX, [(-1., 0.), (1., 0.), (-1., 2.)]),
-    (0., 2., 0., -TY, [(0., -1.), (2., -1.), (0., 1.)]),
-    (0., 2., 0., TXY, [(2., 3.), (4., 3.), (2., 5.)]),
-    (-ROT90, 2., 0., T0, [(0., 0.), (0., -2.), (2., 0.)]),
-    (-ROT90, 2., 0., TX, [(1., 0.), (1., -2.), (3., 0.)]),
-    (-ROT90, 2., 0., TY, [(0., 1.), (0., -1.), (2., 1.)]),
-    (-ROT90, 2., 0., -TX, [(-1., 0.), (-1., -2.), (1., 0.)]),
-    (-ROT90, 2., 0., -TY, [(0., -1.), (0., -3.), (2., -1.)]),
-    (-ROT90, 2., 0., TXY, [(2., 3.), (2., 1.), (4., 3.)]),
-    (ROT90, 2., 0., T0, [(0., 0.), (0., 2.), (-2., 0.)]),
-    (ROT90, 2., 0., TX, [(1., 0.), (1., 2.), (-1., 0.)]),
-    (ROT90, 2., 0., TY, [(0., 1.), (0., 3.), (-2., 1.)]),
-    (ROT90, 2., 0., -TX, [(-1., 0.), (-1., 2.), (-3., 0.)]),
-    (ROT90, 2., 0., -TY, [(0., -1.), (0., 1.), (-2., -1.)]),
-    (ROT90, 2., 0., TXY, [(2., 3.), (2., 5.), (0., 3.)]),
-    (ROT180, 2., 0., T0, [(0., 0.), (-2., 0.), (0., -2.)]),
-    (ROT180, 2., 0., TX, [(1., 0.), (-1., 0.), (1., -2.)]),
-    (ROT180, 2., 0., TY, [(0., 1.), (-2., 1.), (0., -1.)]),
-    (ROT180, 2., 0., -TX, [(-1., 0.), (-3., 0.), (-1., -2.)]),
-    (ROT180, 2., 0., -TY, [(0., -1.), (-2., -1.), (0., -3.)]),
-    (ROT180, 2., 0., TXY, [(2., 3.), (0., 3.), (2., 1.)]),
+    (ROT45, (SQ2, SQ2), 0., T0, [(0., 0.), (1., 1.), (-1., 1.)]),
```

Comment:
Why did you decide to make the scale a tuple for the `SIMILARITY_KNOWN_VALUES`?

Reply:
Because it's also used by the other transformations too. That's why in the actual similarity tests I explicitly change the tuple to a single value.